nerfstudio/viewer/server/viewer_utils.py

```diff
@@ -760,8 +661,12 @@ def _send_output_to_viewer(self, outputs: Dict[str, Any], colors: torch.Tensor =
             self.vis["renderingState/colormap_choice"].write(self.prev_colormap_type)
             self.vis["renderingState/colormap_options"].write(colormap_options)
         selected_output = (self._apply_colormap(outputs, colors) * 255).type(torch.uint8)
-        image = selected_output.cpu().numpy()
-        self.set_image(image)
+
+        image = selected_output[..., [2, 1, 0]].cpu().numpy()
+
+        data = cv2.imencode(".jpg", image)[1].tobytes()
```

Comment:
Maybe the quality could be dropped? The cv2 default of 95 seems quite high, `imageio` has a default of 75:
```
cv2.imencode('.jpg', image, [cv2.IMWRITE_JPEG_QUALITY, 75])
```

`webp` is also an option, file sizes will be smaller but I'd guess encoding will be slightly slower (either the way the render speed seems like it would be the bottleneck though):
```
cv2.imencode(".webp", image, [cv2.IMWRITE_WEBP_QUALITY, 80])
```

Reply:
Dropping to quality 75 decreased the memory ~2x. 
webp is only ~1.15x smaller, but takes ~10x more compute time